keras_cv/losses/iou_loss_test.py

```diff
@@ -12,31 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import numpy as np
 import tensorflow as tf
 
 from keras_cv.losses.iou_loss import IoULoss
 
 
 class IoUTest(tf.test.TestCase):
     def test_output_shape(self):
-        y_true = tf.random.uniform(
-            shape=(2, 2, 4), minval=0, maxval=10, dtype=tf.int32
-        )
-        y_pred = tf.random.uniform(
-            shape=(2, 2, 4), minval=0, maxval=20, dtype=tf.int32
-        )
+        y_true = np.random.uniform(size=(2, 2, 4), low=0, high=10)
+        y_pred = np.random.uniform(size=(2, 2, 4), low=0, high=20)
```

Comment:
Not related to the port so feel free to ignore but it would be good to also seed these tests.

Reply:
What do we get from seeding tests where the assertions are just about shape?

Reply:
Only a suggestion to make the tests reproducible.

In very rare cases, for some random inputs, the test might fail making it sporadically failing through our CI runs. Seeding tests is just a way to avoid those issues. But if we are confident enough that the functions being tested shouldn't do something like that, we can keep them as-is.

Reply:
Ack -- these should be resilient to any inputs within the confines of the random distribution, so I think it's fine as-is, but do let me know if you find other places where we'd be better off seeding tests 😄

Reply:
Sounds good, thanks for the clarification!